src/category_theory/products/default.lean

```diff
@@ -142,18 +151,22 @@ def prod (F : A ⥤ B) (G : C ⥤ D) : A × C ⥤ B × D :=
 /- Because of limitations in Lean 3's handling of notations, we do not setup a notation `F × G`.
    You can use `F.prod G` as a "poor man's infix", or just write `functor.prod F G`. -/
 
-@[simp] lemma prod_obj (F : A ⥤ B) (G : C ⥤ D) (a : A) (c : C) : (F.prod G).obj (a, c) = (F.obj a, G.obj c) := rfl
-@[simp] lemma prod_map (F : A ⥤ B) (G : C ⥤ D) {a a' : A} {c c' : C} (f : (a, c) ⟶ (a', c')) : (F.prod G).map f = (F.map f.1, G.map f.2) := rfl
+@[simp] lemma prod_obj (F : A ⥤ B) (G : C ⥤ D) (a : A) (c : C) :
+  (F.prod G).obj (a, c) = (F.obj a, G.obj c) := rfl
+@[simp] lemma prod_map (F : A ⥤ B) (G : C ⥤ D) {a a' : A} {c c' : C} (f : (a, c) ⟶ (a', c')) :
+  (F.prod G).map f = (F.map f.1, G.map f.2) := rfl
 end functor
 
 namespace nat_trans
 
 /-- The cartesian product of two natural transformations. -/
 def prod {F G : A ⥤ B} {H I : C ⥤ D} (α : F ⟶ G) (β : H ⟶ I) : F.prod H ⟶ G.prod I :=
 { app         := λ X, (α.app X.1, β.app X.2),
-  naturality' := begin /- `obviously'` says: -/ intros, cases f, cases Y, cases X, dsimp at *, simp, split, rw naturality, rw naturality end }
+  naturality' := λ X Y f,
+  begin cases f, cases Y, cases X, simp, split, rw naturality, rw naturality end }
```

Comment:
```suggestion
  by { cases f, cases Y, cases X, simp, split, rw naturality, rw naturality } }
```
I think the idea was to use `by` if it's a 1-line tactic proof.
Also, there is a non-terminal simp in this proof.

Reply:
Fixed. Once I `squeeze_simp`d it wasn't a 1-liner anymore.